### ISSUE TITLE
Missing compatibility code with Multiple Unit Train Control

### ISSUE THREAD

**Issue by robot256** [magu5026/ElectricTrain#7]
Hi magu5026,

Remember back when we added support for ET in MUTC, you had to include the "*-mu" entities in some of your regular expressions so they would be treated like normal electric locomotives?  Looks like there are some similar changes that are needed in the new "control station" code you released last month.  Specifically, the current version of your code throws this error when any version of MUTC is installed, and you have `et-electric-locomotive-1-mu` and `et-electricity-provider` entities on the map at the same time:

```
The mod Electric Train caused a non-recoverable error.
Please report this error to the mod author.

Error while running event ElectricTrain::on_nth_tick(2)
Unknown entity name: et-electric-locomotive-1-mu-power
stack traceback:
	__ElectricTrain__/control.lua:156: in function 'CreateProvider'
	__ElectricTrain__/control.lua:177: in function <__ElectricTrain__/control.lua:172>
stack traceback:
	[C]: in function 'create_entity'
	__ElectricTrain__/control.lua:156: in function 'CreateProvider'
	__ElectricTrain__/control.lua:177: in function <__ElectricTrain__/control.lua:172>
```

Let me know if I can be of any assistance.
-robot256

**Comment by robot256** [comment]
Also noticed that ET version 0.17.14 throws a different error.  I changed my mod to require ET 0.17.12 or .13.  Let me know if you are interested in working out a way to support ET in MUTC that does not require so much testing on your side, I've done it with other mods.

**Comment by robot256** [comment]
Fixed in ET 0.17.28.
